ivy/functional/backends/paddle/layers.py

```diff
@@ -494,3 +494,17 @@ def conv_general_transpose(
     if data_format == "channel_last":
         res = res.transpose(0, *range(2, dims + 2), 1)
     return res
+
+
+# def roi_align(input, boxes, output_size, spatial_scale = 1.0,
```

Comment:
let's remove the commented out implementation as we're deferring to the compositional one

Reply:
yes right, will do. Thanks!